DisplayCAL/setup.py

```diff
@@ -478,11 +478,11 @@ def setup():
             pass
         try:
             import setuptools
-            from setuptools import setup, Extension
+            from setuptools import setup, Extension, find_packages
 
             setuptools = True
             print("using setuptools")
-            current_findall = setuptools.findall
+            current_findall = find_packages
```

Comment:
#214, `findall` was removed/replaced with `find_packages` if my understanding is correct